draft-ietf-core-target-attr.md

```diff
@@ -6,7 +6,7 @@ title: >
 abbrev: CoRE Target Attributes Registry
 docname: draft-ietf-core-target-attr-latest
 # date: 2023-03-01
-updates: 9176
+
```

Comment:
I didn't know that changing one spec's registry from another spec wasn't updating the former spec. I'd like to leave this in so it can trigger the right discussions.